scripts/paper/generate_stwo_phase71_handoff_receipt_figure.py

```
@@ -0,0 +1,584 @@
+        try:
+            rsvg = subprocess.run(
+                ["rsvg-convert", "-f", "png", "-o", str(tmp_png), str(svg_path)],
+                check=False,
+                capture_output=True,
+                text=True,
+            )
+        except FileNotFoundError:
+            tmp_png.unlink(missing_ok=True)
+            raise SystemExit(
+                "rsvg-convert is required to render {} from {}".format(png_path, svg_path)
+            )
```

Ruff (0.15.11)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)